### DISCUSSION TITLE
low cat emoji in the title bar

### DISCUSSION THREAD

**Discussion by zhaob1n** [kovidgoyal/kitty#6471] [Q&A]
the cat emoji is relatively low in the title bar. How can I fix this?
I am using fedora 38, Gnome 44 and wayland 
this font is Hack Nerd Font Mono
![Screenshot from 2023-07-16 06-10-39](https://github.com/kovidgoyal/kitty/assets/52552971/f966a39f-5b79-467b-ae28-dfeb91938894)
![Screenshot from 2023-07-16 06-10-07](https://github.com/kovidgoyal/kitty/assets/52552971/c27dcf2c-a1d3-47ed-bf3f-eda7588581ba)

**Comment by kovidgoyal** [comment]
You cannot it is not a bug.

On Sat, Jul 15, 2023 at 03:16:27PM -0700, ykpcx wrote:
> the cat emoji is small in the title bar. How can I fix this?
> I am using fedora 38, Gnome 44 and wayland 
> this font is Hack Nerd Font Mono
> ![Screenshot from 2023-07-16 06-10-39](https://github.com/kovidgoyal/kitty/assets/52552971/f966a39f-5b79-467b-ae28-dfeb91938894)
> ![Screenshot from 2023-07-16 06-10-07](https://github.com/kovidgoyal/kitty/assets/52552971/c27dcf2c-a1d3-47ed-bf3f-eda7588581ba)
> 
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/6471
> You are receiving this because you are subscribed to this thread.
> 
> Message ID: ***@***.***>

-- 
_____________________________________

Dr. Kovid Goyal 
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________
